common/persistence/sql/common.go

```diff
@@ -41,23 +41,30 @@ import (
 )
 
 // TODO: Rename all SQL Managers to Stores
-type sqlStore struct {
-	db     sqlplugin.DB
+type SqlStore struct {
```

Comment:
Because visibility code in another package. It is in sub dir and can be think of as sub-package but there is no such concept in go, so I need to just export it. In ideal case this needs to be extracted as separate package and used by both main and visibility stores. I plan to do this refactoring later or delegate it to you.